contracts/USM.sol

```diff
@@ -120,28 +126,30 @@ contract USM is ERC20 {
      * @return Debt ratio.
      */
     function debtRatio() public view returns (uint) {
-        if (address(this).balance == 0) {
+        uint pool = ethPool();
+        if (pool == 0) {
             return 0;
         }
-        return totalSupply().wadDiv(ethToUsm(address(this).balance));
+        return totalSupply().wadDiv(ethToUsm(pool));
     }
 
     /**
      * @notice Calculates the price of FUM using its total supply
      * and ETH buffer
      */
-    function fumPrice() public view returns (uint) {
+    function fumPrice(int side) public view returns (uint) {
```

Comment:
Using an `enum` you can skip the `require`, as the function won't accept values that haven't been defined.

`function fumPrice(Side side) public view returns (uint)`

Reply:
👍